libs/backend/src/cast_vote_records/export.ts

```diff
@@ -110,6 +108,32 @@ interface ExportContext {
   usbMountPoint: string;
 }
 
+/**
+ * A sheet to be included in a CVR export
+ */
+export interface ResultSheet {
```

Comment:
Moved this from `legacy_export.ts` to `export.ts` so that `export.ts` doesn't have to import from `legacy_export.ts`. A precursor to fully deleting `legacy_export.ts` in an upcoming PR